### ISSUE TITLE
Add CSS Validator?

### ISSUE THREAD

**Issue by hatzipanis** [renyard/validity#73]
Have you considered adding W3C's CSS Validator also? Much in the same was as [Unicorn](https://validator.w3.org/unicorn/) works?

**Comment by renyard** [comment]
Thanks for raising this.

Yes, it is something I considered in the early days of Validity, but never implemented it as I considered HTML validation more important than CSS validation, due to CSS having more defined error handling and the widespread use of vendor prefixed rules and other pre-standardisation features that would have produced a lot of noise.

Those arguments are several years out of date now, as things have changed since then, so it is something that I'd be more inclined to add now. With that said, with the widespread use of tools like Gulp & Grunt, a build step may be a more suitable place to do CSS validation/linting for many users.

**Comment by hatzipanis** [comment]
You're not wrong that it might be better placed in a build tool, however I know that for our team it would be equally beneficial to have this available in the browser console as well.
